wandb/sdk/wandb_login.py

```diff
@@ -276,6 +276,7 @@ def _login(
     referrer: str = "models",
     _silent: Optional[bool] = None,
     _disable_warning: Optional[bool] = None,
+    _update_api_key: bool = True,
```

Comment:
Parameters shouldn't start with an underscore (these other ones did because of janky `locals()` usage in the past, but the names should be updated).